docs/DeveloperGuide.md

```diff
@@ -242,6 +243,49 @@ Step 3. The parser will construct the `DeleteTagCommand` differently based on th
 
 Step 4. During `DeleteTagCommand#execute()`, the force flag is checked. If present, it unassigns all `Person` with the `Tag` tag1 before deleting tag1.
 
+### Tag Feature
+
+The Tag Feature allows users to store additional details of a Person in WedLinker.
+Tags would support the following functions:
+
+* `Create Tag` — Creates a `Tag` in WedLinker to be used as additional information of any `Person`.
+* `Tag` — Assigns `Tag` to a `Person.
+* `Untag` — Unassigns a `Tag` from a `Person`.
+* `Delete Tag` — Deletes the `Tag` from WedLinker.
+
+#### Implementation
+
+Given below is an example usage scenario and how Tags are used in WedLinker.
+
+Step 1. The user launches the application, `Tags` are loaded into the `Model`.
+
+Step 2. The user executes `create-tag t/Photographer`. WedLinker will create a Tag based on the name provided. In this case: `Photographer`.
+
+Step 3. The user executes `tag 1 t/Photographer` to tag the first `Person` with `Photographer`.
+
+Step 4. To view the Tags in WedLinker, use `list-tags`. This will switch the right panel to a Tag view.
+
+Step 5. The user executes `find t/Photographer` to find all `Person` that has the tag of `Photographer`.
+
+Step 6. The user executes `untag 1 t/Photographer`. WedLinker would remove the `Photographer` tag from the first person.
+
+Step 7. The user executes `delete-tag t/Photographer`. WedLinker would delete the `Photographer` tag.
+
+<box type="info" seamless>
+Tag supports the force functionality for easier usage.
+
+Force is supported for the following functions:
+- `Tag` (Creates the Tag if it does not already exist in WedLinker.)
+- `Delete Tag` (Unassigns all Person from the Tag before it is deleted.)
+  </box>
+
+<box type="warning" seamless>
+Known bugs:
+- Tag names have a limited length and would overflow in the People view. No issues to functionality. 
```

Comment:
Same here with the 'would', and also I think maybe is incorrect to say that tag names have a limited length, it's just the UI that limits how much of the tag can be shown. Maybe more accurate to say 'Tag names with very long names overflow in the People view. No issues...'?